pybliometrics/scopus/serial_title.py

```diff
@@ -180,11 +199,11 @@ def __str__(self):
             f"'{self.publisher}', is active in {areas}\n"
         metrics = []
         if self.sjrlist:
-            metrics.append(f"SJR ({self.sjrlist[0]}): ({self.sjrlist[1]})")
+            metrics.append(f"SJR: {self.sjrlist}")
```

Comment:
What kind of string will this produce? Will it be a list turned to a string? That would look strange

Reply:
Indeed. I will add each year item as a string to the metrics list